build.gradle.kts

```
@@ -7,6 +7,8 @@ buildscript {
     dependencies {
         classpath(deps.plugins.android)
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.72")
+        classpath("com.android.tools.build:gradle:4.2.0-alpha04") //Added due to compatibility issues
+        classpath( "com.squareup.sqldelight:gradle-plugin:1.4.0") //sqlDelight plugin
```

You can use the `deps.plugins` value you added for this instead of having the string itself. We can also omit the comments here
```suggestion
        classpath(deps.plugins.sqldelight)
```